pkg/util/dtversion/dtversion.go

```diff
@@ -48,3 +50,14 @@ func ToSemver(version string) (string, error) {
 
 	return result, nil
 }
+
+func ToImageTag(version string) string {
```

Comment:
@0sewa0 could you please add unit-test to this function?

Reply:
Question: This function is rather dumb, as it doesn't do any verification, should I add a verification?

The problem with that is that I would need to modify the signature of the function, and the signature of most functions that use it. (shouldn't be that problematic)

Reply:
you can remove/refactor or change it :) just noticed it doesn't have tests...

Reply:
done